File: beta/rust/src/help.rs
// src/help.rs

pub fn display_help() {
    println!("Help Information:");
    println!("Usage:");
    println!("  main <command> [options]");
    println!();
    println!("Commands:");
    println!("  run <C file>       Compile and run the specified C file.");
    println!("  run -r <C file>    Compile the specified C file without running it.");
    println!("  -h                 Display this help message.");
    println!("  -v                 Display the version information.");
}

File: stable/go/src/help.go
// help.go
package main

import "fmt"

// DisplayHelp prints the usage instructions and available commands.
func DisplayHelp() {
	fmt.Println("Usage: codersh <command>")
	fmt.Println("Commands:")
	fmt.Println("  run         - Runs a c program and removes output binary")
	fmt.Println("  run -r      - Compiles an c program into binary only.")
	fmt.Println("  --v         - Gives version")
	fmt.Println("  --h         - Gives some help")
}
